inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -514,6 +515,13 @@ study_data_cleaned <- study_data_cleaned %>%
 # (e.g. any rows with end date before start date)
 check_dates <- assert_positive_timespan(study_data_cleaned, startdate, enddate)
 
+# create a variable for month of end (used for deaths by month)
```

Comment:
@zkamvar is there a more efficient way to do this months stuff?

Reply:
I don’t even recall working with this part, tbh.

Reply:
i added it yesterday!